configure.go

```diff
@@ -116,8 +117,11 @@ func Configure(configStruct interface{}) error {
 				valueField.SetString(*flagStringValueMap[tagVal])
 			}
 		case reflect.Bool:
-			if *flagBoolValueMap[tagVal] {
-				flagFound = true
+			if flagBoolValueMap[tagVal] != nil {
+				// we can only know if a bool flag was set if the default was changed
+				if *flagBoolValueMap[tagVal] != config.Field(i).Bool() {
```

Comment:
Should we just fold these expressions into the one `if` boolean expression?

Reply:
https://github.com/Clever/configure/pull/9/commits/016083ff03437b4ecef77fdfe91fa20c9c8168e0